Formula/proteinortho.rb

```diff
@@ -0,0 +1,22 @@
+class Proteinortho < Formula
+  desc "Detecting orthologous genes within different species"
+  homepage "https://gitlab.com/paulklemm_PHD/proteinortho"
+  url "https://gitlab.com/paulklemm_PHD/proteinortho/-/archive/v6.0/proteinortho-v6.0.tar.gz"
+  sha256 "e82522cdae62c182fc3bb75903e8ce3a388573fccbea16b72a0a8c47a653a34b"
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "lapack"
+  depends_on "gcc" # for gfortran
+
```

Comment:
```suggestion

  fails_with :clang # needs fortran

```

Reply:
Then it fails with 
No package 'lapack' found
also i need `pkg-config` to find lapack

Reply:
Hi guys, I'm a bit confused here: do the C/C++ portions of the code fail with Apple clang? If so, why? "needs Fortran" is not a valid reason for adding a `fails_with` block...

Generally, when formulae require Fortran, you add a `depends_on "gcc"` and that is sufficient. All C code is generally built with Apple Clang, and Fortran with the latest GCC.

Also, if you need pkg-config to find lapack in a homebrew formula, for a CMake based project then, you're doing something wrong somewhere...

Reply:
I agree with this assessment. Compile the Fortran code with `gfotran` and the C++ code with `clang++`.